doc/developer/design/20241008_expression_cache.md

```diff
@@ -72,6 +72,7 @@ struct Expressions {
     physical_plan: DataflowDescription<mz_compute_types::plan::Plan>,
     dataflow_metainfos: DataflowMetainfo<Arc<OptimizerNotice>>,
     notices: SmallVec<[Arc<OptimizerNotice>; 4]>,
+    optimizer_feature_overrides: OptimizerFeatureOverrides,
```

Comment:
I think this should be `OptimizerFeatures`. (`OptimizerFeatureOverrides` changes features compared to the system config, but the system config is what can get changed from LD or `ALTER SYSTEM`. See e.g. [here](https://github.com/MaterializeInc/materialize/blob/c4a74b9872bc753162b47315940ea0379a1fecc0/src/adapter/src/coord/sequencer/inner/create_materialized_view.rs#L249-L251).)

Reply:
Just updated to use `OptimizerFeatures`.